README.md

```diff
@@ -1,4 +1,4 @@
-![alt text](https://cdn.discordapp.com/attachments/387632118613671936/822452175694594078/wetgr.png)
+![alt text](https://cdn.discordapp.com/attachments/387632118613671936/822453280981909524/3rt4e.png)
```

Comment:
Change alt text to something else like Zeppelin Banner or something!

Reply:
"Oh no the banned failed to load, guess I'll look at the alt text... Oh... It says alt text... Hmm..."

:mmlol: